Assets/Scripts/Bullet/BulletService.cs

```diff
@@ -23,18 +23,21 @@ public BulletController GetBullet(TankType tankType)
             {
                 BulletModel bulletModel = new BulletModel(bulletScriptableObjectList.Bullets[0]);
                 BulletController bulletController = bulletControllerPoolService.GetBulletController(bulletModel, bulletView);
+                bulletController.BulletView.gameObject.SetActive(true);

```

Comment:
bullet view / game object should not be visible outside the bullet controller. 
Abstraction - you need to hide away the details internal to the controller

Reply:
Abstracted in the bullet pooler service itself as if getBulletItem is not null set active true